mars/storage/tests/test_libs.py

```diff
@@ -175,6 +175,16 @@ async def test_base_operations(ray_start_regular, storage_context):
     np.testing.assert_array_equal(get_data3.toarray(), s1.A)
     np.testing.assert_array_equal(get_data3.todense(), s1.A)
 
+
```

Comment:
Note that I split the `reader/writer` test into two separated case, as vineyard's `reader/writer` doesn't use the default seriliazation.